cmd/poller/plugin/README.md

```diff
@@ -111,6 +113,68 @@ The plugin tries to intelligently aggregate metrics based on a few rules:
 - **Weighted Average** - applied if metric has property `average` and suffix `_latency` and if there is a matching `_ops` metric. (This is currently only matching to ZapiPerf metrics, which use the Property field of metrics.)
 - **Ignore** - metrics created by some plugins, such as value_to_num by LabelAgent
 
+# Max
+
+Max creates a new collection of metrics (Matrix) by calculating max of metric values from an existing Matrix for a given label. For example, if the collected metrics are for volumes, you can create max at nodes or svms level.
```

Comment:
, you can create a max metric at the node or svm level.

Reply:
might also be a good idea to show that example or change the text to match one of the existing examples